### DISCUSSION TITLE
Github Actions: Error: Can't deploy more than one path.

### DISCUSSION THREAD

**Discussion by chrizzly2309** [vercel/turborepo#9615] [Help]
### Summary

# Vercel CLI Multiple Paths Error in Turborepo Setup

## Issue
When trying to deploy using Vercel CLI in a Turborepo setup, receiving the error:

## Project Structure
- Turborepo with Next.js app in `apps/web`
- Project is using `bun` for package management

## Deployment Command
```bash
vercel build --token=${{ env.VERCEL_TOKEN }}
vercel deploy --prebuilt --token=*** 
```

in my vercel project, I have `apps/web` as the root dir. 

During `vercel deploy`, I get the error `Error: Can't deploy more than one path.`

Anybody having had this issue before and resolved it? It's probably a misconfig within my vercel project

**Comment by anthonyshew** [comment]
It might depend how you've linked the project. Are you using `vc link --repo`?: https://vercel.com/docs/cli/link#repo-alpha
